uat/testing-features/src/main/resources/greengrass/components/recipes/client_java_sdk.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: '2020-01-25'
+ComponentName: aws.greengrass.client.Mqtt5JavaSdkClient
+ComponentVersion: '1.0.0'
+ComponentDescription: MQTT5 Java Client powered by AWS Java SDK
+ComponentPublisher: Amazon
+ComponentConfiguration:
+  DefaultConfiguration:
+    agentId: agent1
+    broker:
+      address: 127.0.0.1
+      port: 47619
+Manifests:
+  - Artifacts:
+      - URI: classpath:/greengrass/artifacts/aws.greengrass.client.Mqtt5JavaSdkClient.jar
+        Permission:
+          Read: ALL
+          Execute: ALL
+    Lifecycle:
+      Run: |
+        java -jar {artifacts:path}/aws.greengrass.client.Mqtt5JavaSdkClient.jar {configuration:/agentId} {configuration:/broker/address} "{configuration:/broker/port}"
```

Comment:
Let's use `Startup` and use Greengrass IPC to report that the component is in the running state. https://docs.aws.amazon.com/greengrass/v2/developerguide/ipc-component-lifecycle.html#ipc-operation-updatestate

Reply:
Changed to use Startup 369a951b8f6c1ab252d97cacf06c211091282565

Reply:
Could you clarify about IPC usage, please? Would you like to change "Then the Greengrass deployment is COMPLETED on the device after 180 seconds" step implementation?

Reply:
No, I do not want to change the implementation of the deployment. I'm suggesting that the implementation Mqtt5JavaSdkClient.jar should use Greengrass IPC to report that it is in the running state when it is running.